project/Dependencies.scala

```diff
@@ -66,9 +66,10 @@ object Dependencies {
   val databaseDependencies = List(
     "com.typesafe.slick" %% "slick" % slickV,
     "com.typesafe.slick" %% "slick-hikaricp" % slickV,
-    "org.hsqldb" % "hsqldb" % "2.3.2",
+    "org.hsqldb" % "hsqldb" % "2.3.4",
     "mysql" % "mysql-connector-java" % "5.1.36",
-    "com.github.pathikrit" %% "better-files" % betterFilesV % Test
+    "com.github.pathikrit" %% "better-files" % betterFilesV % Test,
+    "org.broadinstitute" %% "wdl4s" % wdl4sV // Used in migration scripts
```

Comment:
If you all want to move towards libraries that are militant about pure FP and typeclasses I'd support that ;)